cpplite/cpplite.editor/nbproject/org-netbeans-modules-cpplite-editor.sig

```diff
@@ -24,7 +24,9 @@ meth public abstract void removeChangeListener(javax.swing.event.ChangeListener)
 CLSS public static org.netbeans.modules.cpplite.editor.spi.CProjectConfigurationProvider$ProjectConfiguration
```

Comment:
I guess `ProjectConfiguration` inner class shouldn't be part of API to begin with, @jlahoda?